src/main/java/TaskList.java

```diff
@@ -0,0 +1,85 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TaskList class.
+ * Keeps the tasks as a list of this instance.
+ *
+ * @author Amos Cheong
+ */
+public class TaskList {
+
+    private List<Task> listOfTask;
```

Comment:
You could just call it tasks. Not sure if it is a convention or just something my tutor said, but he said that you should usually name lists as the plural of the item the list contains.